### DISCUSSION TITLE
Lanesto 322054 - device not recognized correctly

### DISCUSSION THREAD

**Discussion by christiaangombert** [Koenkk/zigbee2mqtt#23600] [Devices]
Hi, I have a Lanesto 322054 dimmable led driver. This device is listed on the z2m website as supported, see: [https://www.zigbee2mqtt.io/devices/322054.html](url)

Sadly my z2m installation does not recongnize the device correctly, see attached screenshot image. 
![Schermafbeelding 2024-08-12 220415](https://github.com/user-attachments/assets/efe6946a-cf44-4c7d-a81d-639bcdd96521)

How can a fix this? I'm new to z2m, so sorry if this sounds newby.

**Comment by Koenkk** [comment]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Comment by skoehorst** [comment]
> Hi, I have a Lanesto 322054 dimmable led driver. This device is listed on the z2m website as supported, see: [https://www.zigbee2mqtt.io/devices/322054.html](url)
> 
> Sadly my z2m installation does not recongnize the device correctly, see attached screenshot image. ![Schermafbeelding 2024-08-12 220415](https://private-user-images.githubusercontent.com/20791713/357186713-efe6946a-cf44-4c7d-a81d-639bcdd96521.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2ODc1NjAsIm5iZiI6MTcyNzY4NzI2MCwicGF0aCI6Ii8yMDc5MTcxMy8zNTcxODY3MTMtZWZlNjk0NmEtY2Y0NC00YzdkLWE4MWQtNjM5YmNkZDk2NTIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTMwVDA5MDc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyNjMzMmRkNTNlMGMwNmQ4NjA4MmI2NGNjNzJiYzE0NWY5NTYzYzZjOGU2M2VmZWM4YzM4YTE3NTVkNmNhMGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.U2YQxrSuoDNeM0Z9NIxPg00ffuqmaWyufVKqZdDWcAc)
> 
> How can a fix this? I'm new to z2m, so sorry if this sounds newby.

Can you tell me how you even got this found? I’ve got the same but am unable to find it…

**Reply by christiaangombert** [reply]
Finding the device was not easy for me. To set this device into pairing mode, this worked for me:

Connect the wires of the led-driver, leds and zigbee-dongle.

Then do the following routine 5 times in a row:
- disconnect the zigbee-dongle from the led-driver.
- wait 3 sec.
- connect the zigbee-dongle to the led-driver.
- wait 3 sec.

Set zigbee2mqtt in pairing/discovery mode. Hope for the best and you'll find the device. I took me 2 or 3 times to get it right.

For more info, see chapter 3 in this Dutch manual:
[https://www.dekkerzevenhuizen.nl/amfile/file/download/file/84/product/2038/](url)
